egs2/TEMPLATE/asr1/scripts/utils/TEMPLATE_Readme.md

```diff
@@ -0,0 +1,50 @@
+---
+tags:
+- espnet
+- audio
+- <add_more_tags>
+language: <add_lang>
+datasets:
+- <add_corpus>
+license: cc-by-4.0
+---
+## ESPnet2 <add_task_name> pretrained model 
+### `<add_model_name>`
+♻️ Imported from <add_url>
+
+This model was trained by <add_name> using <add_corpus>/<add_recipe_task_name> recipe in [espnet](https://github.com/espnet/espnet/).
+### Demo: How to use in ESPnet2
+```python
+# coming soon
+```
+### Citing ESPnet
```

Comment:
For this citation, ideally, I want to change it depending on the recipe type, e.g., 

- watanabe2018espnet for all types and ASR
- hayashi2020espnet for TTS
- li2021espnet for SE
- inaguma2020espnet for ST

@article{inaguma2020espnet,
  title={ESPnet-ST: All-in-one speech translation toolkit},
  author={Inaguma, Hirofumi and Kiyono, Shun and Duh, Kevin and Karita, Shigeki and Soplin, Nelson Enrique Yalta and Hayashi, Tomoki and Watanabe, Shinji},
  journal={arXiv preprint arXiv:2004.10234},
  year={2020}
}

@inproceedings{li2021espnet,
  title={ESPnet-SE: end-to-end speech enhancement and separation toolkit designed for ASR integration},
  author={Li, Chenda and Shi, Jing and Zhang, Wangyou and Subramanian, Aswin Shanmugam and Chang, Xuankai and Kamo, Naoyuki and Hira, Moto and Hayashi, Tomoki and Boeddeker, Christoph and Chen, Zhuo and others},
  booktitle={2021 IEEE Spoken Language Technology Workshop (SLT)},
  pages={785--792},
  year={2021},
  organization={IEEE}
}

Reply:
I have made changes of citation for ASR, TTS and SE

Reply:
For the speaker diarization, we may not have to add it but just add watanabe2018espnet for now.